modules/ROOT/pages/manage-exchange-api-task.adoc

```diff
@@ -34,10 +34,16 @@ Use this option if you want to provide additional security for your API.
 +
 For example, you can use the URL of your API asset in Exchange. In this case, the API is consuming the JSONPlaceholder API, so enter `+http://jsonplaceholder.typicode.com+`.
 +
-. If you are creating an endpoint with proxy, accept the default path `/` for the proxy endpoint in the *Path* field, or enter another path. 
+. If you are creating an endpoint with proxy, accept the base path `/` for the proxy endpoint in the *Path* field, or enter another path. 
 +
-For example, changing the path to `/users` makes the REST call to the service at the implementation URI `+http://jsonplaceholder.typicode.com/users+`.
+For example, if an API is defined with the `Implementation URI` set to `http://jsonplaceholder.typicode.com` the proxy returns the following URIs based on the value you configure in the proxy path:
```

Comment:
`http://jsonplaceholder.typicode.com` the proxy > `http://jsonplaceholder.typicode.com`**,** the proxy

Reply:
Good catch! Missed that. THanks!